### PR TITLE
prometheus-jitsi-exporter: init at 0.2.18

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

I use this to monitor my Jitsi meet instance.
https://grafana.petabyte.dev/d/GWCJOoPMz/jitsi-meet?orgId=1&refresh=10s&from=now-7d&to=now

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mmilata** [comment]
Does the exporter export only these two metrics, conferences and participants? There's [another one](https://github.com/karrieretutor/jitsi-prom-exporter) which seems to export more [detailed data](https://github.com/karrieretutor/jitsi-prom-exporter/blob/master/examples/example-metrics.md).

**Comment by petabyteboy** [comment]
No, it exports a lot more: https://mozarella.petabyte.dev/jitsi-exporter/metrics

**Review by mmilata — APPROVED** [review]
Did not test but the package and module look good.

File: pkgs/servers/monitoring/prometheus/jitsi-exporter.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  };
+
+  vendorSha256 = null;
+
+  meta = with lib; {
+    description = "Export Jitsi Videobridge metrics to Prometheus";
+    homepage = "https://git.xsfx.dev/prometheus/jitsiexporter";
+    license = licenses.mit;
+    maintainers = with maintainers; [ petabyteboy ];
+    platforms = platforms.unix;
```
Comment (SuperSandro2000) [inline pkgs/servers/monitoring/prometheus/jitsi-exporter.nix:20]:
```suggestion
```

set by buildGoModule

**Review by WilliButz — APPROVED** [review]

**Pull request merged**